NOTES.md

```diff
@@ -1,3 +1,143 @@
+# Release 2.2.0 (Modern Times)
+
+Released on 2015-11-XX
+
+Release highlights:
+
+* Atomic changefeeds
+* Performance & Scalability improvements
+
+Read the [blog post][2.2-release] for more details.
```

Comment:
Sounds good, thanks!
I slightly modified the wording, so you might want to double-check that it still sounds good to you @mglukhovsky .